docs/guide/maven-plugin/goals/index.adoc

```diff
@@ -7,11 +7,13 @@
 |<<_run>> |Run a WildFly bootable JAR (blocking).
 |<<_shutdown>> |Shutdown a WildFly bootable JAR.
 |<<_start>> |Start a WildFly bootable JAR (non blocking).
+|<< help >> |Prints help for the plugin.
```

Comment:
The help needs to be prefixed by _, perhaps add it after dev one?

Reply:
Done